docs/containers.conf.5.md

```diff
@@ -301,6 +301,9 @@ create new containers and pods in that namespace.  The default namespace is "",
  which corresponds to no namespace. When no namespace is set, all containers
 and pods are visible.
 
+**network_cmd_path**=""
+  NetworkCmdPath is the path to the slirp4netns binary.
```

Comment:
Nit: I would prefer
```suggestion
  network_cmd_path is the path to the slirp4netns binary.
```
or
```suggestion
  Path to the slirp4netns binary.
```
for consistency's sake?

Reply:
This is a back port,  You need to fix these in the master branch.  Please open a PR>